packages/aws-amplify-angular/src/components/authenticator/forgot-password-component/forgot-password.component.core.ts

```diff
@@ -116,15 +116,15 @@ export class ForgotPasswordComponentCore {
     this._authState = data.authState;
     this._show = data.authState.state === 'forgotPassword';
 
-    this.username = data.authState.user? data.authState.user.username || '' : '';
+    this.username = data.authState.user && data.authState.user.username ? data.authState.user.username : '';
```

Comment:
```suggestion
    this.username = (data.authState.user && data.authState.user.username) ? data.authState.user.username : '';
```

Just to make it more clear